src/main/java/org/testng/annotations/Parameters.java

```diff
@@ -14,7 +13,7 @@
  * @author <a href="mailto:cedric&#64;beust.com">Cedric Beust</a>
  */
 @Retention(RUNTIME)
-@Target({METHOD, CONSTRUCTOR, TYPE })
+@Target({METHOD, TYPE})
 public @interface Parameters {
```

Comment:
I cannot identify any added value to test and to give constructor's parameters unitary.
I see your test example, but, I prefer to test a setter method instead of constructor to avoid nested and indirect tests.

Thks,